### ISSUE TITLE
pdf.worker.js include-path is wrong when pdf.js is initiated from a web sub-dir

### ISSUE THREAD

**Issue by cgmurray** [jjalvarezl/JjalvarezlPDFjsViewerBundle#4]
Hi, nice job on this library. It works great and saves us Symfony-heads a lot of effort. I tried to fix this issue myself but couldn't figure it out, due to the varying paths and includes based on dev/test vs. production, as well as the bower build dist versions, and not knowing which version your bundle is based on.
I'm on a shared, and very controlled hosting platform, and must use your installer to install to a sub-directory when pushing to production. The installer does all the right things except one, the JS that calls pdf.worker.js assumes: /bundles/jjalv..., but it needs to be: /mySubDir/bundles/jjalva...
I can fix it in dev with a symlink, but I can't do that in production because my app is self-contained in its own sub-directory.
Is there any possibility that this is an easy fix, or you could provide some clues on how to fix it? Thx!

**Comment by cjsfj** [comment]
I'm also affected by this. Good explanation.

**Comment by jjalvarezl** [comment]
Yes, it is right, I have not tested yet, and I do not have enough time to solve the bugs, but I'm going to develop them. If you already solved them please let me know or make a pull request to the current version. Thank you for using my solution.
